README.md

```diff
@@ -756,7 +756,15 @@ Grafana, pgAdmin. Additionally, downloads a memcached and Elastic as project dep
 
 1. Sign up with any fictional account at [sign_up page](http://www.rubygems.example:8000/sign_up) of the app. That credentials will be used later when uploading gem.
 
-2. The application we are deploying is gem storage, so we have to build gem that we will use in testing.
+2. Verify your email on [special page](http://www.rubygems.example:8000/rails/mailers/mailer/email_confirmation) for development purposes
```

Comment:
so the readme says "sign up with fictional account", but then "verify email". so the email should be real?

Reply:
no, you can pass anything you want and the next link for verification specially for test emails to avoid real verification. It provided for development purposes by rubygems project.

Reply:
actually, it is not our feature and instructions. all people familiar with that project maybe know or can read the original [documentation](https://github.com/rubygems/rubygems.org/blob/master/CONTRIBUTING.md#confirmation-emails-links)

Reply:
i understand, just suggesting how to make people's life easier when they are reading the instructions